packages/cascara/src/modules/DataTextArea/DataTextArea.doc.mdx

```diff
@@ -24,7 +24,6 @@ Feel free to play around with the following code:
   <DataTextArea
     attribute='firstName'
     isEditable={true}
-    isLabeled={true}
```

Comment:
@lordpixel in the scenario of documentation if we are showing what is possible on a module, setting a prop on a component should be okay, even if that prop is something that we do not let end-users set.

The problem area is if we are trying to set isLabeled in a module _object_. Those are the ones we should get away from.

@CaffeinatedAllie just a heads up that the actual docs might not show a label for modules but the label will indeed be set correctly from the 'label' prop.